src/main/scala/mesosphere/marathon/api/v2/DeploymentsResource.scala

```diff
@@ -28,7 +28,10 @@ class DeploymentsResource @Inject() (
   @GET
   def running(@Context req: HttpServletRequest): Response = authenticated(req) { implicit identity =>
     val infos = result(service.listRunningDeployments())
-      .filter(_.plan.affectedApplications.exists(isAuthorized(ViewRunSpec, _)))
+      .filter(steps => (steps.plan.affectedApplications.isEmpty ||
```

Comment:
I added an `affectedRunSpecs` in my branch on deployments which hold all apps and pods that are affected.